src/transformers/models/decision_transformer/configuration_decision_transformer.py

```diff
@@ -79,7 +81,7 @@ class DecisionTransformerConfig(PretrainedConfig):
             The standard deviation of the truncated_normal_initializer for initializing all weight matrices.
         scale_attn_weights (`bool`, *optional*, defaults to `True`):
             Scale attention weights by dividing by sqrt(hidden_size)..
-        use_cache (`bool`, *optional*, defaults to `True`):
+        use_cache (`bool`, *optiofrom typing import Mappingnal*, defaults to `True`):
```

Comment:
It looks like a random copy/paste text here :)

Reply:
fixed